Lib/test/test_signal.py

```diff
@@ -1251,6 +1252,44 @@ def handler(signum, frame):
         # Python handler
         self.assertEqual(len(sigs), N, "Some signals were lost")
 
+    def test_stress_modifying_handlers(self):
+        # bpo-43406: race condition between trip_signal() and signal.signal
+        signum = signal.SIGUSR1
```

Comment:
Skip this test if there's no `signal.SIGUSR1` (i.e. Windows).

Reply:
Will do, thanks.